theme.css

```diff
@@ -315,19 +325,19 @@ body:not(.doc-title-disable) .inline-title {
 body.doc-title-accent .inline-title {
   background-image: linear-gradient(
     var(--gradientDegree),
-    hsl(calc(var(--accent-h) + 30), 100%, 75.1%) 0,
-    hsl(calc(var(--accent-h) - 30), 100%, 75.1%) 100%
+    hsl(calc(var(--accent-h) + 30), var(--core-lightness), var(--core-saturation)) 0,
+    hsl(calc(var(--accent-h) - 30), var(--core-lightness), var(--core-saturation)) 100%
```

Comment:
this commit seems to break the whole UI when selecting "match headings" option. "Accent Color" does not work.

Reply:
you're right, these are the wrong way around. I'll push a fix shortly

Reply:
fixed in: https://github.com/caro401/royal-velvet/pull/28/commits/5d0c495743e782cf0326755202a2f5a945a48232